index.html

```diff
@@ -115,6 +115,37 @@ <h2>A project for first-time contributions</h2>
         <!-- make sure there is one line of space above and below your card -->
         <!-- ========== ⬇⬇ Paste YOUR CARD directly BELOW this line ⬇⬇ ========== -->
 
+        <!-- ________ Tetiana Stetsyk card START ________  -->
+        <div class="card">
+          <p class="name">Tetiana Stetsyk</p>
+          <p class="contact">
+            <i class="fab fa-facebook"></i>
+            <a href="https://www.facebook.com/tetyana.ivanochko/" target="_blank">tetianastetsyk</a>
+          </p>
+          <p class="about">I'm starting my career in AQA</p>
+          <div class="resources">
+            <p>3 Useful Dev Resources</p>
+            <ul>
+              <li>
+                <a href="https://testautomationu.applitools.com/" target="_blank" title="Test Automation University">
+                  Test Automation University
+                </a>
+              </li>
+              <li>
+                <a href="https://www.youtube.com/c/AutomationStepByStep" target="_blank" title="Automation Step by Step">
+                  Automation Step by Step
+                </a>
+              </li>
+              <li>
+                <a href="https://www.youtube.com/c/Freecodecamp" target="_blank" title="Free Code Camp">
+                  FreeCodeCamp
+                </a>
+              </li>
+            </ul>
+          </div>
+        </div>
+        <!-- ________*END TEMPLATE* Contributor card END ________  -->
```

Comment:
It looks like you forgot to change this comment. But no worries.
I'll change it.